plugins/meson.build

```diff
@@ -36,6 +35,7 @@ endif
 
 if get_option('gudev') and get_option('gusb')
 subdir('logitech-hidpp')
+subdir('system76-launch')
```

Comment:
I think technically `gudev` isn't needed for this, only `gusb`.  So it should be a separate `get_option` block

Reply:
Okay, I will change the patch accordingly